docs/tutorials/create-first-spore/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 1
+---
+
+import EmbedCard from "@site/src/components/EmbedCard";
+
+# Creating your first spore
+
+Imagine having complete ownership and control over your digital content, stored securely on a decentralized blockchain, without relying on external URLs or third-party platforms. That's the power of Spore Protocol!
+
+In this tutorial, we'll walk you through how to create your very first spore from scratch.
+
+Before we begin, let's take a quick look at this example:
+
+<EmbedCard
+  title="Spore: 0x075b3210...6061ba7493 | Spore Demo"
+  href="https://a-simple-demo.spore.pro/spore/0x075b32100cbc6c082b4c80cd2e2713c30ae0eb2f494dddff435d206061ba7493"
+  className="margin-top--sm margin-bottom--lg"
+/>
+
+This spore, hosted on the Nervos CKB Testnet, stores an image directly on the blockchain. With Spore Protocol, you have *full* control over the *entire* content, not just a URL to where the content is stored often off-chain.
+
+If this piques your interest, let's dive into this step-by-step guide to:
+
+1. [Set up an account on Nervos CKB](./setup-account)
+2. [Create your first spore on-chain](./create-spore)
+3. [Explore the code](./explore-code)
+
+Let's begin by creating an account on Nervos CKB: [Set up an account on Nervos CKB](./setup-account).
```

Comment:
![Screenshot 2023-10-18 at 12 02 53 AM](https://github.com/sporeprotocol/spore-docs/assets/112425356/d68d236f-74e1-476d-b0eb-94733ab03ff1)
having both is redundant?

Reply:
Do you think we should remove the first one?